frontend/src/libs/components/button/button.tsx

```diff
@@ -48,6 +52,11 @@ const Button: React.FC<Properties> = ({
     small: styles.buttonSmall,
   };
 
+  const outlineColorClassNameMapper: Record<ButtonOutlineColor, string> = {
+    green: styles.green,
+    white: styles.white,
+  };
```

Comment:
As of now keys are directly matching the classes - so we can pass them directly I guess